.github/pr-custom-review.yml

```diff
@@ -0,0 +1,18 @@
+rules:
```

Comment:
It was unclear if my comment https://github.com/paritytech/substrate/pull/10951/files#r817289492 got unnoticed so I'll suggest again.

After https://github.com/paritytech/pr-custom-review/pull/57 has landed, you could do it like this:

```diff
  - name: Source files
    check_type: changed_files
-   condition: .*
+   condition:
+     exclude: ^(ci|\.maintain|\.github)/
    any:
      - min_approvals: 2
        teams:
          - core-devs
      - min_approvals: 1
        teams:
          - substrateteamleads
+
+ - name: CI files
+   check_type: changed_files
+   condition: ^(ci|\.maintain|\.github)/
+   min_approvals: 2
+   teams:
+     - ci
```

Are you interested in this approach?

Reply:
Thanks for the suggestion. But for now its not a priority. Main goal is to focus on most important rules.